doc: Add known issue for CIS Peripheral upon central reset (#14595)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

.. rst-class:: v2-6-0

DRGN-21619: The controller might assert if the CIS peripheral stops receiving packets from the CIS central
  This only occurs when the window widening reaches at least half of the ISO interval in magnitude.

>   Assuming worst case clock accuracies on both the central and the peripheral, this could occur with a supervision timeout of 2.4 s, 3.7 s, and 4.9 s and an ISO interval of 5 ms, 7.5 ms, and 10 ms, respectively.

Reviewer: ```suggestion
  Assuming worst case clock accuracies on both the central and the peripheral, this could occur with a supervision timeout of 2.4, 3.7, or 4.9 seconds, corresponding to an ISO interval of 5, 7.5, or 10 milliseconds, respectively.
```
